crm/modules/member/page.inc.php

```diff
@@ -108,13 +108,13 @@ function member_page (&$page_data, $page_name, $options) {
             }
             // Add view tab
             $view_content = '';
-            if (user_id() == $_GET['cid'] || ((user_access('contact_edit') && user_access('member_edit')))) {
+            if (user_id() == $cid || ((user_access('contact_edit') && user_access('member_edit')))) {
```

Comment:
Simple fix. Already pulled it from the GET array on line 105.  Saves at least a couple CPU cycles to pull it directly from a scalar variable instead of looking it up in an array again.